src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.stories.tsx

```diff
@@ -26,12 +24,7 @@ export default {
 };
 
 export const ConnectedComponent = () => {
-  return (
-    <NoDataViews
-      onDataViewCreated={action('onDataViewCreated')}
-      dataViewsDocLink={docLinksServiceFactory().dataViewsDocsLink}
```

Comment:
Why this change?

Reply:
The connected component [requires](https://github.com/elastic/kibana/blob/main/src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.tsx#L19) this link.  Our plugin has integrated the docs service, and the example component in Storybook was [calling](https://github.com/elastic/kibana/blob/main/src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.stories.tsx#L32) the `docLinks` service to populate the property.  So it appeared as though consumers of this connected component would have to either 1/ use doc links themselves in their plugin, or 2/ use our service.  Either way, it seemed unnecessary to make a consumer populate this prop when we have access to it through our connected services.